src/seedu/addressbook/data/person/Street.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+public class Street {
+	
+	private String value;
+	
+	/**
+	 * Constructor with initial street name
+	 */
+	Street(String initialValue) {
+		this.value = initialValue;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.